apis/v1alpha1/ack-generate-metadata.yaml

```diff
@@ -1,8 +1,8 @@
 ack_generate_info:
-  build_date: "2022-12-08T19:01:42Z"
-  build_hash: 12246c7da82841b351ec7a9e1f139f9338f2784b
-  go_version: go1.17.5
-  version: v0.21.0
+  build_date: "2022-12-13T20:52:55Z"
+  build_hash: 39d7c30b8df3c44a9804f578cd6acc9a3f669b03
+  go_version: go1.19
+  version: v0.20.1-15-g39d7c30-dirty
```

Comment:
nit: make sure that you're generating the controller with a non dirty version

Reply:
Done